src/vm/amd64/jithelpers_fastwritebarriers.S

```diff
@@ -244,3 +246,331 @@ PATCH_LABEL JIT_WriteBarrier_SVR64_PatchLabel_CardTable
         mov     byte ptr [rdi + rax], 0FFh
         ret
 LEAF_END_MARKED JIT_WriteBarrier_SVR64, _TEXT
+
+#endif
+
+
+#ifdef FEATURE_USE_SOFTWARE_WRITE_WATCH_FOR_GC_HEAP
+
```

Comment:
Are we gaining much by having all the different optimized flavors of the write barrier with software write watch? It is pretty non-trivial to test them all...